app/samples/tenants_and_tasks/prj.conf

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 RIT Launch Initiative
+# SPDX-License-Identifier: Apache-2.0
+
+# CPP
+CONFIG_CPP=y
+CONFIG_REQUIRES_FULL_LIBCPP=y
+CONFIG_STD_CPP20=y
+
+CONFIG_F_CORE=y
+
+# OS
+CONFIG_POLL=y
+CONFIG_MAIN_STACK_SIZE=1024
+CONFIG_STACK_USAGE=y
+
+CONFIG_TIMESLICING=y
+CONFIG_TIMESLICE_SIZE=50000
+CONFIG_TIMESLICE_PRIORITY=0
+CONFIG_THREAD_STACK_INFO=y
+
+CONFIG_DYNAMIC_THREAD=y
+# Max stack size for dynamic threads
+CONFIG_DYNAMIC_THREAD_STACK_SIZE=512
+# Number of threads in the dynamic thread pool
+# (Zephyr wants 4 for some reason instead of 2)
```

Comment:
What does this mean specifically. Is it always 2x what you actually get?

Reply:
So we should theoretically specify 2 since we specify 2 threads, but this fails out. When this program runs theres 5 threads. Counter, Print, sysworkq, shell_uart, idle and main. For some reason we just need 4 though. The other 3 threads that Zephyr defines shouldn't need dynamic mem which is super weird